src/applications/edu-benefits/0994/pages/highTechIndustry.js

```diff
@@ -8,6 +8,77 @@ const {
   currentSalary,
 } = fullSchema.properties;
 
+const HIGH_TECH_EMPLOYMENT_TYPES = {
```

Comment:
I don't think this is necessary.  In the form configs we don't usually worry about duplicate property in both `uiSchema` and `schema` names and consolidating them

Reply:
It may be overkill to some degree, but I don't like having string literals repeated in the same file subject to typos and not extensible.